packages/amplify-cli-logger/src/IAmplifyLogger.ts

```diff
@@ -0,0 +1,8 @@
+import { LogErrorPayload, LogPayload } from './Types';
+
+export interface IAmplifyLogger {
+  log(message: string): void;
```

Comment:
How is `log` different from `logInfo`? Is there something more descriptive?

Reply:
it's left over from plugging in the logger directly to aws-sdk removed it now